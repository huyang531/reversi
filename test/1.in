cd
cc
cb
ce
cf
be
ef
df
ae
bd
ad
dc
eg
dg
cg
af
db
ch
bf
ac
bc
fh
bh
ah
eh
dh
ag
bg
ab
aa
bb
ca
ba
da
ea
fg
ec
ff
fe
gf
hf
hg
hh
he
ge
fd
fc
eb
gd
hd
gb
fb
gc
fa
gg
hc
ga
hb
ha
gh
