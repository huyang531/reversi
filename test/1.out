  abcdefgh
a ........
b ........
c ........
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c ...B....
d ...BB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c ..WB....
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c .BBB....
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ........
c .BBBW...
d ...WW...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ........
c .BBBBB..
d ...WB...
e ...BW...
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ....W...
c .BBBWB..
d ...WW...
e ...BW...
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ........
b ....W...
c .BBBWB..
d ...WB...
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ........
b ....W...
c .BBBWB..
d ...WWW..
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ....B...
b ....B...
c .BBBBB..
d ...WBW..
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ....B...
b ...WB...
c .BBWWB..
d ...WBW..
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ...BB...
b ...BB...
c .BBBWB..
d ...BBW..
e ...BBB..
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ...BB...
b ...BB...
c .BBBWB..
d ..WWWW..
e ...BBB..
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ...BB...
b ...BB...
c .BBBBB..
d ..WWWB..
e ...BBBB.
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ...BB...
b ...BB...
c .BBBBB..
d ..WWWWW.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ...BB...
b ...BB...
c .BBBBBB.
d ..WWWBB.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ...BBW..
b ...BW...
c .BBWBBB.
d ..WWWBB.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ...BBW..
b ...BW...
c .BBWBBB.
d .BBBBBB.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ...BBW..
b ...BW...
c .BBWWWWW
d .BBBBBB.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ...BBW..
b ...BBB..
c .BBWBBWW
d .BBBBBB.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ...BBB..
c .BBWBBWW
d .BBBBBB.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..BBBB..
c .BBBBBWW
d .BBBBBB.
e ...BBBB.
f ........
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW..
b ..BWBB..
c .BBBWBWW
d .BBBBWB.
e ...BBBW.
f .......W
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW..
b ..BWBB.B
c .BBBWBBW
d .BBBBBB.
e ...BBBW.
f .......W
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW.W
b ..BWBB.W
c .BBBWBBW
d .BBBBBB.
e ...BBBW.
f .......W
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..WWWW.W
b ..BWBB.W
c .BBBWBBW
d .BBBBBB.
e ...BBBBB
f .......W
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..WWWW.W
b ..BWBW.W
c .BBBWBWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a ..WWWWBW
b ..BWBB.W
c .BBBBBWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a ..WWWWBW
b ..BWWWWW
c .BBBBBWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a .BBBBBBW
b ..BWWWWW
c .BBBBBWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b ..BWWWWW
c .BBBBBWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b .WWWWWWW
c .BBBBBWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b .WWWWWWW
c WWWWWWWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c WBWWWWWW
d .BBBBBBW
e ...BBBBW
f .......W
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c WWWWWWWW
d WWWWWWWW
e ...BBBBW
f .......W
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e B..BBBBW
f .......W
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e B..BBWWW
f ......WW
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BBWWWWWW
e B.BBBWWW
f ......WW
g ........
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BBWWWWWW
e B.BBWWWW
f .....WWW
g ........
h ........
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BBWWWWWW
e B.BWWWWW
f ....WWWW
g ........
h ........
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBBWWWWW
e B.BBWWWW
f ....BWWW
g .....B..
h ........
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBBWWWWW
e B.BBWWWW
f ....BWWW
g .....W..
h .....W..
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBBWWWWW
e B.BBWWWW
f ....BWWW
g .....B..
h .....WB.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBBWWWWW
e B.BBWWWW
f ....BWWW
g .....B..
h .....WWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBBWWWWW
e B.BBWWWW
f ....BWWW
g .....W..
h ....WWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBBWWWWW
e B.BBWWWW
f ....WWWW
g ....WW..
h ....WWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBBWWWWW
e B.BWWWWW
f ...WWWWW
g ....WW..
h ....WWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BBWWWWWW
d BBWWWWWW
e B.WWWWWW
f ..WWWWWW
g ....WW..
h ....WWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e BWWWWWWW
f ..WWWWWW
g ....WW..
h ....WWWW
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e BBWWWWWW
f ..BWWWWW
g ...BWW..
h ....WWWW
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e BBWWWWWW
f ..BWWWWW
g ...WWW..
h ...WWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e BBWWWWWW
f ..WWWWWW
g .W.WWW..
h ...WWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e BWWWWWWW
f .WWWWWWW
g .W.WWW..
h ...WWWWW
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b BWWWWWWW
c BWWWWWWW
d BWWWWWWW
e BWWWWWWW
f .BWWWWWW
g .WBWWW..
h ...WWWWW
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g .WBWWW..
h ...WWWWW
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g .WBBBBB.
h ...WWWWW
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g .WWWBBB.
h ..WWWWWW
Enter move for colour B (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g BBBBBBB.
h ..WWWWWW
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g BWWBBBB.
h .WWWWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g WWWBBBB.
h WWWWWWWW
B player has no valid move.
Enter move for colour W (RowCol):   abcdefgh
a WWWWWWWW
b WWWWWWWW
c WWWWWWWW
d WWWWWWWW
e WWWWWWWW
f WWWWWWWW
g WWWWWWWW
h WWWWWWWW
B player has no valid move.
W player has no valid move.
White wins by 64 points!
